src/config/NetworkConfig.ts

```diff
@@ -149,10 +149,13 @@ export const defaultMainnetNetworkConfig: NetworkConfig = {
         { friendlyName: 'ngl-dual-605', roles: 2, url: 'http://ngl-dual-605.symbolblockchain.io:3000' },
     ],
 };
-
-const defaultNetworkConfig: Record<number, NetworkConfig> = {
-    152: defaultTestnetNetworkConfig,
-    104: defaultMainnetNetworkConfig,
+export const defaultGenerationHashes: Record<string, string> = {
```

Comment:
Every time the defaultGenerationHashes is used, it's a code smell. The wallet is indirectly indexing by network type anyway.  network type -> default hash -> some data